webhook_server/tests/test_github_api.py

```
@@ -871,7 +871,7 @@ async def test_get_pull_request_github_exception(
 
     @pytest.mark.asyncio
     async def test_get_pull_request_by_commit_with_pulls(
-        self, minimal_hook_data: dict, minimal_headers: dict, logger: Mock
+        self, minimal_hook_data: dict, minimal_headers: Headers, logger: Mock
```

Ruff (0.14.11)

Unused method argument: `minimal_hook_data`

(ARG002)